.github/workflows/regenerate-stateful-test-disks.yml

```diff
@@ -40,7 +47,7 @@ jobs:
           --container-image rust:buster \
```

Comment:
Should we search and replace all instances of `buster` with `bullseye`?

Reply:
Here is a bit different, as this is the container's Host. But I'm planning to use the built image directly, instead of adding a host, then ssh into it, and then run docker inside. I'll cut all those steps and run the container with different entrypoints/CMD to run the needed commands.